nilearn/input_data/nifti_labels_masker.py

```diff
@@ -214,16 +214,14 @@ def transform(self, imgs, confounds=None):
                 self._resampled_labels_img_ = self.labels_img_
             if not _check_same_fov(imgs, self._resampled_labels_img_):
                 logger.log("resampling labels", verbose=self.verbose)
-                self._resampled_labels_img_ = self._cache(image.resample_img,
-                    func_memory_level=1)(
+                self._resampled_labels_img_ = self._cache(image.resample_img, 2)(
```

Comment:
Any reason why you changed func_memory_level in this file? Used to be 1 now is 2.